monkey/infection_monkey/master/ip_scanner.py

```diff
@@ -95,6 +95,8 @@ def _run_fingerprinters(
         fingerprint_data = {}
 
         for f in interruptable_iter(fingerprinters, stop):
```

Comment:
At least use `finger` abreviation. `f` usually stands for `function`. Even if it's a small function bad readability is bad readability, not everyone reads code top to bottom, one line at a time. I usually skip the loop definition because it's not important to track the call stack, but then I need to go back to understand what "f" is.

Reply:
"The length of a variable name should be proportional to its scope." ([source](https://twitter.com/unclebobmartin/status/39333982050975744?lang=en))

> At least use `finger` abreviation.

`finger` is something you have on your hand. It doesn't actually mean anything in this context. `for fingerprinter in fingerprinters:` is awkward and could lead to confusion. You need to really pay attention to what is being manipulated.

>  `f` usually stands for function

I usually use `fn`

> I usually skip the loop definition...

Given that this loop is one line long, it seems to me that it would be pretty difficult to skip the loop definition.

Do you have a better recommendation for a name?

Reply:
> finger is something you have on your hand. It doesn't actually mean anything in this context.
And `f` is a letter of an alphabet if you choose this point of view and also means nothing :D Both are abbreviations, one is simply longer.

> Given that this loop is one line long, it seems to me that it would be pretty difficult to skip the loop definition.
It's easy. Even easier, because in this review the loop definition was not modified, so it doesn't glow green. 

Personally, I would vote for never using one-letter abbreviations, except for universal conventions like `i` for iterator, `x` for lambda arg, etc. But I understand that it's a minor thing and a personal preference, so keep it.

Reply:
> And f is a letter of an alphabet if you choose this point of view and also means nothing :D Both are abbreviations, one is simply longer.

Exactly. `finger` is more typing to also be meaningless. If I'm going to be meaningless, at least I can save some keystrokes.  :p

Reply:
> Personally, I would vote for never using one-letter abbreviations

That's fine. I generally agree, although in tight loops it doesn't bother me. 

I'm not opposed to a better name for `f`, I just don't think `finger` is a better name. I could even see it being misleading in some contrived scenario.